CHANGES.rst

```diff
@@ -1,3 +1,13 @@
+v7.1.0
```

Comment:
I'm not sure whether this counts as a breaking change or a feature. Maybe patch?

Reply:
I guess technically it's a breaking change as it requires more recent versions of setuptools to install from source. I'll update to 8.0.